CMakeLists.txt

```diff
@@ -185,16 +185,18 @@ LINK_DIRECTORIES(${Boost_LIBRARY_DIRS})
 # up the targets (Exported) that it needs. We just need to find where
 # HDF5 is installed.
 
-if("${HDF5_INSTALL}" STREQUAL "")
-    set(HDF5_INSTALL  $ENV{HDF5_INSTALL})
-endif()
+find_package(HDF5)
```

Comment:
We need to leave the check for the HDF5_INSTALL before the "find_package()" as the "find_package()" uses that variable. If it is blank then HDF5 will not be found correctly.